### ISSUE TITLE
Browser support missing

### ISSUE THREAD

**Issue by kubikaugustyn** [mdn/content#33781]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers

### What specific section or headline is this issue about?

"Common container formats" and "Browser compatibility"

### What information was incorrect, unhelpful, or incomplete?

In all of the browser support tables (e.g. `Video codecs supported by 3GP`, `Audio codecs supported by 3GP` and all of the others) there is only `Firefox` support set to `Yes` in some places and nothing more, e.g. the tables are blank.

### What did you expect to see?

I expected to see the browser support fields filled with information about - well, whether the browser supports the format or not.

### Do you have anything more you want to share?

![image](https://github.com/mdn/content/assets/47718385/8c9d0485-43d4-455c-961b-843be9eb03f1)
![image](https://github.com/mdn/content/assets/47718385/bb78d2fb-0c2b-4adc-88d4-9f1208488a03)
![image](https://github.com/mdn/content/assets/47718385/44f97395-8901-49e8-9391-974afca74df3)
![image](https://github.com/mdn/content/assets/47718385/affd9daf-da42-4840-bbe1-29bc7ea3ba70)
and more blank browser support tables...

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/media/formats/containers`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/media/formats/containers/index.md
* Last commit: https://github.com/mdn/content/commit/e83bbd923268ae2b664a79f8931b78f4206061db
* Document last modified: 2024-04-03T08:41:12.000Z

</details>

**Comment by hamishwillee** [comment]
You're absolutely correct. It is this way because we don't know what is supported and what is not, and have no easy way to find out. There is more information about Firefox because Mozilla started this documentation and have populated most of the parts that are supported.

Do you have additional information that we can use to populate the tables?

**Comment by kubikaugustyn** [comment]
Sadly, I don't have any additional information about the browser support. I was actually looking on MDN to find out which audio and video codecs are supported by all browsers.

If files encoded with all those codecs are gathered into one HTML file (into video and audio tags), they can be all tested at once by opening the HTML file in all of these browsers.

**Comment by Josh-Cena** [comment]
Work paused until https://github.com/mdn/browser-compat-data/issues/6971
